audiomate/corpus/conversion/base.py

```diff
@@ -49,9 +50,10 @@ def convert(self, corpus, target_audio_path):
         out_corpus = audiomate.Corpus()
         files_to_convert = []
 
-        print('Find utterances to convert ...')
```

Comment:
I'd keep all those but use `logging.info(msg, *args, **kwargs)` or maybe `logging.debug(msg, *args, **kwargs)`. This would help with tracing/logging. And the user has control.

Reply:
I removed that one because it is part of the ``progress`` function. Otherwise there is duplicate output.